power_systems_data_api_demonstrator/src/lib/db/dao/grid_node_dao.py

```diff
@@ -46,7 +46,9 @@ async def add_generation(
             self.session.add(gen)
         await self.session.commit()
 
-    async def get_all_grid_nodes(self, limit: int | None = None) -> List[GridNodeModel]:
+    async def get_all_grid_nodes(
+        self, limit: Optional[int] = None
```

Comment:
This must be a compatibility thing, the `int | None` syntax is valid starting in python3.10. https://peps.python.org/pep-0604/

Reply:
This PR should resolve this issue: https://github.com/carbon-data-specification/Power-Systems-Data-API-Demonstrator/pull/32

Reply:
Ok let's wait for #32 to be fixed and re-asses :)